slides/src/content/redux-philosophy.md

```diff
@@ -0,0 +1,60 @@
+# Redux Philosophy
+
+---
+
+## Motivation 
+
+1. Create [Single Source of Truth](https://en.wikipedia.org/wiki/Single_source_of_truth) 
+1. Reduce Complexity of Mental Map
+1. Create a coherent framework so future developers will know how to add features by following convention
+1. Promote separation of responsibility between components
+1. Improve logic flow by having things _react_ to events as opposed to caller dictating actions
+1. Functional Philosophy : Separation of Data and Logic
+
+---
+
+## Structure of Systems - Redux vs Server Architecture
+
+| Function | Server   | Angular Redux  |
```

Comment:
I think the order of the columns could be inverse: "In Redux, a **store** is like a **database** that provides **data storage**". Seems more natural to me to include terms in that order.